### DISCUSSION TITLE
Git commit template 사용법

### DISCUSSION THREAD

**Discussion by bentshrimp** [AgainIoT/Open-Set-Go#3] [Announcements]
# Git Commit Template 적용

```bash
git config --local commit.template .gitmessage.txt
```

# :memo: commit 남기는 방법

```bash
git commit
```

```vi
<타입> : <제목> #<이슈넘버>

<본문>
```

# commit 타입

| 타입     | 내용                                                                 |
| -------- | -------------------------------------------------------------------- |
| Init     | 개발 환경 초기 세팅                                                  |
| Feat     | 새로운 기능 추가                                                     |
| Fix      | 버그 수정                                                            |
| Update   | Fix와 달리 원래 정상적으로 동작했지만 보완의 개념                    |
| Remove   | 파일을 삭제하는 경우                                                 |
| Move     | 코드나 파일을 이동하는 경우                                          |
| Rename   | 파일 혹은 폴더명을 수정하는 경우                                     |
| Docs     | 문서를 수정한 경우                                                   |
| Comment  | 필요한 주석 추가 및 변경                                             |
| Refactor | 코드 리팩토링 (결과의 변경 없이 코드의 구조를 재조정, 가독성을 높임) |
| Test     | 테스트 코드                                                          |
| Chore    | 그 외 자잘한 수정                                                    |

**Comment by bentshrimp** [comment]
나온대로 커밋 템플릿을 적용하면
```bash
git commit
```
을 실행해 commit을 작성할 때마다 템플릿을 보면서 즐겁게 코딩할 수 있답니다~ :D

**Comment by ymw0407** [comment]
You can see this style guide in English also on our Contributing.md!
https://github.com/AgainIoT/Open-Set-Go/blob/main/CONTRIBUTING.md#commit-message-style
